var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -82,7 +82,7 @@ class Xsdk(Package):
     depends_on('sundials@3.1.0~int64+hypre', when='@0.3.0')
     depends_on('sundials@3.1.0~int64+hypre', when='@develop')
 
-    depends_on('plasma@17.1', when='@develop %gcc@4.9:')
+    depends_on('plasma@develop', when='@develop %gcc@6.0:')
```

Comment:
if you know that this will hold for next release, you can write `plasma@17.2:` and Spack will currently choose `develop` but as soon as `17.2` is released, it will choose the stable version.

Reply:
Ah - I was using 17.2 in other parts anyway - so changed to 17.2 here aswell. We can update this later if needed